test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -4,12 +4,35 @@
 import static org.junit.Assert.assertTrue;
 
 import java.util.Arrays;
-import java.util.List;
 
 import org.junit.Test;
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
This is a correctly written test. 
Please also be aware of the 3 part test name format that may be useful when you write tests for your project.